programs/axelar-solana-gas-service/src/lib.rs

```diff
@@ -12,7 +12,10 @@ use solana_program::msg;
 use solana_program::program_error::ProgramError;
 use solana_program::pubkey::Pubkey;
 
+#[cfg(not(feature = "devnet"))]
 solana_program::declare_id!("gasFkyvr4LjK3WwnMGbao3Wzr67F88TmhKmi4ZCXF9K");
+#[cfg(feature = "devnet")]
```

Comment:
shouldn't this be `testnet` ? our commented `'B` :)

So, if we do not specify anything, devnet id is assumed. if we specify `testnet` , then testnet id will be used. In a future, if we specify `mainnet`, mainnet i will be used.